security/advancedtls/examples/credential_reloading_from_files/README.md

```diff
@@ -0,0 +1,22 @@
+# Credential Reloading From Files
+
+Credential reloading is a feature supported in the advancedtls library. 
+A very common way to achieve this is to reload from files.
+
+In this example, users will need to provide a set of locations on the file system holding the credential data.
+Once the credential data needs to be updated, users just need to change the credential data in the file system, and gRPC will pick up the changes automatically.
```

Comment:
The client is making a single connection to the server. Given that established connections are not affected by change in the files being watched, how does this example demonstrate the credential reloading feature?

Reply:
Gotcha. I understand your concern. I think the purpose of this example is to demonstrate how to set the API. The credential reloading feature is covered in `advancedtls_integration_test`, which reloads credentials in temp files. 
This example is needed by that internal project, which will have a link pointed to this example in their user guide. The original sample code they gave me also didn't include the reloading feature.
I think the current wordings in README is a bit confusing. I've updated some text. Could you please take a look? Thank you so much!